packages/eslint-plugin/tests/docs.test.ts

```diff
@@ -180,6 +181,32 @@ describe('Validating rule docs', () => {
           }
         });
       }
+
+      test('must include only valid code samples', () => {
+        for (const token of tokens) {
+          if (token.type !== 'code') {
+            continue;
+          }
+
+          const lang = token.lang?.trim();
+          if (!lang?.startsWith('ts')) {
+            return;
+          }
+
+          try {
+            parseForESLint(token.text, {
+              ecmaFeatures: {
+                jsx: lang.startsWith('tsx'),
+              },
+              ecmaVersion: 'latest',
+              sourceType: 'module',
+              range: true,
```

Comment:
Why this?

Reply:
Without `range: true` parser throws `Cannot read properties of undefined (reading '0')` on valid code